### DISCUSSION TITLE
Regarding Appium Server<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by sg12057** [community/community#131814] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi Team,

I am Using Appium Server application and the version which I am using is 1.22.3-4 , I just want to know like for appium server what are the END-OF-LIFE dates i need EOL dates for appium server please respond to my email address *personal identifying information removed by community moderator*

**Comment by LiteBrite82** [comment]
Hi USERNAME,

Thanks for being a part of the GitHub Community, @sg12057. We're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Appium, repository](https://github.com/appium/appium). 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
